tools/lib/dmod_dependencies/vscode-dmd/example.dmd

```diff
@@ -0,0 +1,29 @@
+# Example DMOD Dependencies File
+# This file demonstrates the .dmd format
+
+# Simple module entries without versions
+dmffs
+driver
+make_dmffs
+
+# Module entries with specific versions
+spi@1.0
+i2c@2.5
+uart@1.2.3
+
+# Include another dependencies file
+$include https://raw.githubusercontent.com/example/deps.dmd
+
+# Change the manifest source for following modules
+from: https://registry.example.com/manifest.dmm
```

Comment:
dla spójności jednak niech to będzie `$from` a nie `from:` - bo inaczej może się mylić... Mimo, że `from:` wydaje się czytelniejsze, ale lepiej żeby to działało poprawnie